test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -441,17 +448,10 @@ def deprecation_schema(for_collection):
     date_schema.update(main_fields)
 
     if for_collection:
-        version_schema = {
-            Required('removed_in'): version(for_collection),
-        }
+        version_schema = {Required('removed_in'): version(for_collection)}
```

Comment:
I'd keep the original style here:

```suggestion
        version_schema = {
            Required('removed_in'): version(for_collection),
        }
```